vital/kwiversys/CommandLineArguments.cxx

```diff
@@ -134,6 +134,19 @@ void CommandLineArguments::Initialize(int argc, char* argv[])
   this->Initialize(argc, static_cast<const char* const*>(argv));
 }
 
+// ----------------------------------------------------------------------------
+void CommandLineArguments::Initialize(const std::vector<std::string>& argv)
```

Comment:
Is this upstream in KWSys?  If not, should it be?

Reply:
This has not yet been pushed upstream.